src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -33,15 +40,32 @@ public AddressBook(ReadOnlyAddressBook toBeCopied) {
         persons.setPersons(toBeCopied.getPersonList());
     }
 
-    //// list overwrite operations
+    //// undo & redo operations
 
     /**
-     * Makes a copy of persons, and stores it in personsList.
+     * Empties the redoList.
      */
-    private void save() {
+    private void emptyRedoList() {
```

Comment:
Would `clearRedoList` be a better name for this method?

Reply:
Yes, it has been changed.